packages/design-system/src/components/slider/slider.tsx

```diff
@@ -0,0 +1,311 @@
+
+  const classNames = useMemo(
+    () => mergeClassNames(sliderClassNames, theme.Slider, classNamesProp),
+    [theme.Slider, classNamesProp],
+  );
+
+  const style = useCallback(
+    (renderProps: SliderTrackRenderProps) =>
+      inlineVars(sliderTrackStateVars, renderProps),
+    [],
+  );
+
+  const values = useMemo<
+    [
+      [
+        typeof SliderThumbContext,
+        ContextValue<SliderThumbProps, HTMLDivElement>,
+      ],
+    ]
+  >(() => [[SliderThumbContext, { classNames }]], [classNames]);
+
+  const children = useCallback(
+    (renderProps: SliderTrackRenderProps) => (
+      <Provider values={values}>
+        {callRenderProps(childrenProp, renderProps)}
+      </Provider>
+    ),
+    [childrenProp, values],
+  );
+
+  return (
+    <RACSliderTrack
+      {...rest}
+      ref={ref}
+      className={classNames?.track?.container}
+      style={style}
+    >
+      {children}
+    </RACSliderTrack>
+  );
+});
+
+/**
+ * SliderThumb must be used as a child of SliderTrack
+ */
+
+export const SliderThumbContext =
+  createContext<ContextValue<SliderThumbProps, HTMLDivElement>>(null);
+
+export const SliderThumb = forwardRef(function SliderThumb(
+  props: SliderThumbProps,
+  ref: ForwardedRef<HTMLDivElement>,
+) {
+  [props, ref] = useContextProps(props, ref, SliderThumbContext);
+
+  props = useDefaultProps(props, 'SliderThumb');
+
+  const { children: childrenProp, classNames: classNamesProp, ...rest } = props;
+
+  const theme = useTheme();
+
+  const classNames = useMemo(
+    () => mergeClassNames(sliderClassNames, theme.Slider, classNamesProp),
+    [theme.Slider, classNamesProp],
+  );
+
+  const style = useCallback(
+    (renderProps: SliderThumbRenderProps) =>
+      inlineVars(sliderThumbStateVars, renderProps),
+    [],
+  );
+
+  const values = useMemo<
+    [[typeof AriaLabelContext, ContextValue<LabelProps, HTMLLabelElement>]]
+  >(() => [[AriaLabelContext, {}]], []);
+
+  const children = useCallback(
+    (renderProps: SliderThumbRenderProps) => (
+      <Provider values={values}>
+        <div className={classNames?.thumb?.thumb}>
+          {callRenderProps(childrenProp, {
+            ...renderProps,
+            orientation: 'horizontal',
+          })}
+        </div>
+      </Provider>
+    ),
+    [childrenProp, values, classNames?.thumb?.thumb],
+  );
+
+  return (
+    <RACSliderThumb
+      {...rest}
+      ref={ref}
+      className={classNames?.thumb?.container}
+      style={style}
+    >
+      {children}
+    </RACSliderThumb>
+  );
+});
+
+export const SliderContext =
+  createContext<ContextValue<SliderProps, HTMLDivElement>>(null);
+
+export const Slider = forwardRef(function Slider(
+  props: SliderProps,
+  ref: ForwardedRef<HTMLDivElement>,
+) {
+  [props, ref] = useContextProps(props, ref, SliderContext);
+
+  props = useDefaultProps(props, 'Slider');
+
+  const {
+    children: childrenProp,
+    classNames: classNamesProp,
+    alignLabel = 'top',
+    minValue = 0,
```

Comment:
It looks like this is unused and because it's destructured it's not being passed along to the RAC